modules/swagger-codegen/src/main/java/io/swagger/codegen/Codegen.java

```diff
@@ -48,6 +48,8 @@ public static void main(String[] args) {
         options.addOption("d", "debug-info", false, "prints additional info for debugging");
         options.addOption("a", "auth", true, "adds authorization headers when fetching the swagger definitions remotely. Pass in a URL-encoded string of name:header with a comma separating multiple values");
         options.addOption("c", "config", true, "location of the configuration file");
+        options.addOption("m", "support", true, "location of the configuration file");
```

Comment:
@ToreJohansson thanks for the PR. I think the description should be updated to "maps templates with file names template1=file1,template2=file2." (likely you copied it from the "c" option one line above)

Reply:
correct :-)